buyer-api/src/operations/registerPayment.js

```diff
@@ -29,18 +30,27 @@ export const registerPayment = async (notarizationRequestId) => {
    * The operation sends the transaction to the Buyer SS and wait for it to respond.
    * Once the signature is responded,
    * the operation will add new job to the transaction queue to send the transaction to the network.
+   * if account's balance in BatPay is less than the amount to pay, register payments queue will be
+   * paused
    */
   // data payload
   const {
     result: {
       notarizationFee: fee, orderId, sellers, lockingKeyHash,
     },
   } = await notarizations.fetch(notarizationRequestId);
+
   const { transactionHash, price } = await dataOrders.fetchByDxId(orderId);
+  const amount = fromWib(price);
+  if (!(await hasEnoughBatPayBalance(batPayId, amount))) {
```

Comment:
```suggestion
  if (!(await hasEnoughBatPayBalance(batPayId, toBN(amount).muln(sellers.length)))) {
```

Reply:
I think with this change, there is no need to modify `blockchain/balance` module